utils/findutils.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="https://apps.0install.net/utils/findutils.xml" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>FindUtils</name>
+  <summary xml:lang="en">FindUtils: find and then operate on files</summary>
+  <description xml:lang="en">`find' is a program which searches a directory tree to find a file or group of files. It walks the directory tree and reports all occurences of a file matching the user's specifications. 
+
+`locate' scans one or more databases of filenames and displays any matches. 
+
+`xargs' builds and executes command lines by gathering together arguments it reads on the standard input. Most often, these arguments are lists of file names generated by `find'
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png"/>
+  <category>System</category>
+  <homepage>https://sourceforge.net/projects/ezwinports/files/</homepage>
+  <needs-terminal/>
+  <group version="4.2.30.1-5" released="2013-06-22" license="GPL v2 (GNU General Public License)">
+    <environment name="BINDIR" insert="bin" />
+    <environment name="LIBEXECDIR" insert="libexec" />
+    <command name="run" path="bin/find.exe" />
+    <command name="locate" path="bin/locate.exe" />
+    <command name="updatedb" path="bin/updatedb">
+      <requires interface="https://apps.0install.net/utils/coreutils.xml">
+        <environment name="PATH" insert="bin" />
+      </requires>
+      <requires interface="https://apps.0install.net/utils/mktemp.xml">
+        <executable-in-path name="mktemp" />
+      </requires>
+      <requires interface="https://apps.0install.net/utils/bash.xml">
+        <executable-in-var name="SHELL" />
+      </requires>
+      <requires interface="/https://apps.0install.net/utils/sed.xml">
```

Comment:
There's a leading `/` in the URL here that needs to be removed.

Reply:
removed